jbpm-designer-backend/src/test/java/org/jbpm/designer/bpmn2/impl/Bpmn2JsonMarshallerTest.java

```diff
@@ -1008,4 +1008,35 @@ public void testBoundaryEventDefinitionRef() throws Exception {
         assertNotNull(endEvent);
     }
 
+    @Test
+    public void testMessageRefNodes() throws Exception {
```

Comment:
@tsurdilo could you please have a look on formatting in this test? Especially `getChildByName` calls seems to be not well formatted.